.vib/zookeeper/goss/goss.yaml

```diff
@@ -0,0 +1,27 @@
+file:
+  /opt/bitnami/zookeeper/logs:
+    exists: true
+    filetype: directory
+    mode: "0775"
+  /opt/bitnami/zookeeper/conf/zoo.cfg:
+    exists: true
+    filetype: file
+    mode: "0644"
+    contains:
+      - {{ printf "/clientPort=.*%s/" .Env.ZOO_PORT_NUMBER }}
+command:
+  user-id-test:
+    exec: if [ "$(id -u)" -eq 0 ]; then exit 1; fi
+    exit-status: 0
+    stdout: []
+    stderr: []
+  zookeeper-version-ok:
+    exec: /opt/bitnami/zookeeper/bin/zkCli.sh -server 127.0.0.1:$ZOO_PORT_NUMBER version | grep "ZooKeeper CLI version" | cut -d- -f1 | grep --quiet $APP_VERSION
+    exit-status: 0
+    stdout: []
+    stderr: []
+  zookeeper-create-znode:
+    exec: /opt/bitnami/zookeeper/bin/zkCli.sh -server 127.0.0.1:2181 create /zk_test my_data
+    exit-status: 0
+    stdout: []
```

Comment:
It would also be interesting checking that stdout contains a confirmation of the creation, such as:

```
Created /zk_test 
```

This is something that we usually do, take a look at Kafka for instance:

https://github.com/bitnami/charts/blob/be2668ac3b3fb7d15211ba6b763f052d74096b02/.vib/kafka/goss/goss.yaml#L6-L7